app/controllers/nyre/projects_controller.rb

```diff
@@ -37,19 +58,19 @@ def subsite_config
     # haaaaaaack to not reproduce templates
     def initialize(*args)
       super(*args)
-      self._prefixes << subsite_config['layout'] + '/projects' 
-      self._prefixes << subsite_config['layout']
+      self._prefixes << 'nyre/projects' 
+      self._prefixes << 'nyre'
```

Comment:
Less dynamically generated than before.  Deliberate change?

(as opposed to using `subsite_key`)

Reply:
I think this is cleaned up now.

Reply:
Thanks!